src/main/java/com/jcabi/aspects/aj/MethodCacher.java

```diff
@@ -351,12 +362,22 @@ public boolean expired() {
         public boolean asyncUpdate() {
             return this.async;
         }
+
+        /**
+         * Soft reference to cached object.
+         * Visible only for testing. Don not use directly.
+         * @return Soft reference to cached object.
+         */
```

Comment:
@nesterov-n we should avoid getters... http://www.yegor256.com/2014/09/16/getters-and-setters-are-evil.html